### PR TITLE
First iteration of Azure SQL VM deployments 

### PR DESCRIPTION
**Pull request by aasimkhan30**
Just creating this PR for early feedback

### CODE REVIEW

File: extensions/resource-deployment/package.json [hunk 1]
```diff
@@ -391,6 +391,338 @@
...
+                          "type": "azure_account",
+                          "required": true
+                        },
+                        {
+                          "type": "azure_locations",
+                          "label": "%azure-sqlvm-azure-location-label%",
+                          "defaultValue": "eastus",
+                          "required": true,
+                          "locationVariableName": "AZDATA_NB_VAR_AZURE_SQLVM_LOCATION",
+                          "displayLocationVariableName": "AZURE_SQLVM_DISPLAY_LOCATION",
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/package.json:436]: Most of the vars start with AZDATA_NB_VAR_AZURE_SQLVM, but not this one. Intended?

File: extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Create Azure SQL Virtual Machine\n",
+                "============================================\n",
+                "\n",
+                "Description\n",
+                "-----------\n",
+                "For more information about other Azure PowerShell options for creating SQL VMs, see the <a href=\"https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sql/virtual-machines-windows-ps-sql-create\">Provisioning guide for SQL Server VMs with Azure PowerShell</a>. See also <a href=\"https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sql/quickstart-sql-vm-create-powershell\">Quickstart guide</a> to creating a SQL Server VM with Azure PowerShell.\n",
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb:31]: Remove en-us from these links

File: extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+                "\n",
+                "\n",
+                "Steps of this procedure include:\n",
+                "1. Connect to Azure subscription\n",
+                "1. Provision resource group for SQL VM migration\n",
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb:36]: Have these all be either 1. or increasing in value - don't mix them

File: extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+                "azdata_cell_guid": "7dbdc638-d6cd-4ebe-9268-3efc18d3c415"
+            },
+            "outputs": [],
+            "execution_count": null
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Set Subscription\r\n",
+                "Below command will open a _Dialouge Box_ with list of subscriptions. Selecting one of those will set that subscription for rest of the commands."
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb:132]: Dialog Box

File: extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb [hunk 5]
```diff
@@ -0,0 +1,431 @@
...
+                "## Connect to Azure Account"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "b06fee5e-355d-47fc-8c1f-41294756cc87"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "run_command('azdata --version', False);"
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb:120]: You don't need azdata right? Should this be `az --version`?

File: extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb [hunk 6]
```diff
@@ -0,0 +1,431 @@
...
+                "## Set variables"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "37db2e50-dcde-4dd5-820c-7dc11212f1eb"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "if \"AZDATA_NB_VAR_AZURE_SQLVM_PASSWORD\" in os.environ:\r\n",
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb:99]: Given that this notebook isn't using azdata at all I'd suggest changing the var names so there isn't any confusion there. I seem to recall the deployment wizard using the prefix to determine what vars to insert into the doc though so you might need to also change the deployment extension code. @alanrenmsft

File: extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb (hunk 2 above)
Comment (kburtram) [reply extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb:31]: Are the any fwlinks or aka.ms links to this page.  That's typically preferred to embedded fixed URLs into product code.

Comment (aasimkhan30) [reply extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb:31]: Yeah, that is an unnessary link. will remove in the next commit

**Comment by alanrenmsft** [comment]
there must be something wrong with this PR, there are 1290 files changed. @aasimkhan30

**Pull request closed without merging**